catalog-info.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'openedx-atlas'
+  description: "An Open edX CLI tool for moving translation files from openedx-translations."
+  links:
+  - url: "https://github.com/openedx/openedx-atlas"
```

Comment:
You also don't need to add a link to the source, that gets added automatically.

Reply:
removed